phpunit.xml.dist

```diff
@@ -6,7 +6,7 @@
 		 xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/6.1/phpunit.xsd"
```

Comment:
Please update here to use `http://schema.phpunit.de/7.5/phpunit.xsd` instead of 6.1

Reply:
updated :heavy_check_mark: